packages/create-turbo/src/index.ts

```diff
@@ -136,28 +137,35 @@ async function run() {
   let relativeProjectDir = path.relative(process.cwd(), projectDir);
   let projectDirIsCurrentDir = relativeProjectDir === "";
   if (!projectDirIsCurrentDir) {
-    if (fse.existsSync(projectDir)) {
+    if (fse.existsSync(projectDir) && fs.readdirSync(projectDir).length !== 0) {
```

Comment:
This section is the only functional change; we now don't error if the target directory is empty.